foodmart-dataset/src/main/resources/druid/foodmart/foodmart-index.json

```diff
@@ -140,7 +140,7 @@
         },
         {
           "name" : "user_unique",
-          "type" : "hyperUnique",
```

Comment:
the wikiticker dataset currently has a hyperUnique metric, which is what we use to test hyperUnique with. This metric was changed to thetaSketch, because currently in calcite this column is not used.